cpp/neuralnet/nninputs.cpp

```diff
@@ -641,6 +641,54 @@ Board SymmetryHelpers::getSymBoard(const Board& board, int symmetry) {
   return symBoard;
 }
 
+void SymmetryHelpers::maskSymmetricDuplicativeLoc(const Board &board, bool *const isSymDupLoc) {
+  fill(isSymDupLoc, isSymDupLoc + Board::MAX_ARR_SIZE, false);
+  vector<int> symTypes;
```

Comment:
Very minor: maybe `validSymmetries` instead of `symTypes`?

Reply:
I like these comments very much. I want to hold very high standards  to do things. Those details and your thinkings and feedbacks are very helpful.